src/Symfony/Bundle/FrameworkBundle/Command/DebugWarmedDeprecationsCommand.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * Show the deprecations generated during the cache warmup.
+ *
+ * @author Hamza Amrouche <hamza.simperfit@gmail.com>
+ *
+ * @final
+ */
+class DebugWarmedDeprecationsCommand extends Command
+{
+    protected static $defaultName = 'debug:warmed-deprecations';
```

Comment:
`debug:deprecations, debug:deprecations:warmup`?

Reply:
debug:deprecations:warmed feels like better so we can create new command for all deprecations or other types of deprecations

Reply:
right, but dont we prefer a single command and have all the info? Otherwise, yeah i'd prefer the `deprecation:` namespace first.

Reply:
It will too much responsability in one command IMHO, I prefer using :warmed to not be confusing.
It we do something that shows all the deprecation we can then later deprecate this command and add an option to the new command